packages/desktop-client/src/i18n.ts

```
@@ -3,8 +3,15 @@ import { initReactI18next } from 'react-i18next';
 import i18n from 'i18next';
 import resourcesToBackend from 'i18next-resources-to-backend';
 
+const languages = import.meta.glob('/locale/*.json');
+
 const loadLanguage = (language: string) => {
-  return import(`./locale/${language}.json`);
+  const path = `/locale/${language}.json`;
+  if (!languages.hasOwnProperty(path)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)